website/docs/core-concepts/workflows/workflows.mdx

```diff
@@ -29,6 +29,37 @@ workflows:
       - command: terraform apply eks/alb-controller -auto-approve
 ```
 
+## Retry Configuration
+
+The `command` section of the workflow schema has been updated to allow for retrying of failed commands. The `retry` section
+accepts the following parameters:
+
+* `max_attempts`: The maximum number of times the command will be retried. The default is `1`.
+* `delay`: The amount of time to delay between retries. The default is `5s`.
+* `backoff_strategy`: The backoff strategy to use. The default is `constant`. The other options are `exponential` and `linear`.
+* `initial_delay`: The initial delay to use when retrying. The default is `5s`.
+* `random_jitter`: Whether to add a random jitter to the delay between retries. The default is `true`.
+* `multiplier`: The multiplier to use when retrying. The default is `2`.
+* `max_elapsed_time`: The maximum time allocated to the command before the retry fails. The default is 30 minutes.
+
+Here is an example of a workflow with retry configuration:
+```
+workflows:
+  eks-up:
+    description: Bring up the eks cluster
+    steps: 
+      - command: terraform apply vpc -auto-approve
+        retry:
+          max_attempts: 3
+          backoff_strategy: exponential # could be exponential, constant and linear
+          initial_delay: 3s # the time used by the backoff_strategy
+          random_jitter: false # the random jitter to be added while retrying.
```

Comment:
Random jitter should be a maximum amount of jitter (float), not a boolean

Reply:
okay updating

Reply:
Done